addons/mrp/models/mrp_production.py

```diff
@@ -792,7 +792,7 @@ def _compute_is_delayed(self):
 
     @api.onchange('qty_producing', 'lot_producing_id')
     def _onchange_producing(self):
-        self._set_qty_producing()
+        self._set_qty_producing(False)
```

Comment:
I don't think you need this parameter because manual_consumption moves will be set picked=True at `pre_button_mark_done()` anyway

Reply:
When we confirm the manufacturing order with zero quantity, it will be marked as an automatic production, and manual consumption movements will be picked in the call to `_set_quantities()` in the if-block after the loop in `pre_button_mark_done()`.
However, if the production order produces a non-zero value, the manual moves will be picked inside the loop itself. The function does the default behavior when when we don't specify the `pick_manual_consumption_moves`. The only place we need it to be `False` is when we set the quantities manually from `_onchange_producing()`